README.md

```diff
@@ -90,7 +90,7 @@ in which case you may not even need the asset pipeline. This is mostly relevant
 
 ## Features
 
-* [Webpack 2](https://webpack.js.org/)
+* [Webpack 2 and 3](https://webpack.js.org/)
```

Comment:
Maybe drop the `2`? We technically don't support it now: https://github.com/rails/webpacker/blob/b4ded385e4cc520cc90ef46d3491ff0927d6ada8/package.json#L37

Reply:
Hmm yeah, although the config we have will work with both versions but makes sense to just say 3. 

Also, we should add that module concatenation plugin? https://github.com/webpack/webpack/tree/master/examples/scope-hoisting#webpackconfigjs

Reply:
Will push an update now 👍

Reply:
The config may work, but you won't be able to install v2 if you have webpacker as a dependency in package.json